modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -567,14 +560,16 @@ use {{invokerPackage}}\ObjectSerializer;
         }
 
         // for model (json/xml)
-        if (isset($_tempBody)) {
-            // $_tempBody is the method argument, if present
+        {{#bodyParams}}
```

Comment:
This one produced nonsensical code unless you have body params:
1. sets `$_tempBody` to null
2. checks if `$_tempBody` is not null

It can never be not null because the code mutating it is not generated, why are the checks for it generated then?